packages/strapi-plugin-content-type-builder/admin/src/translations/es.json

```diff
@@ -24,29 +80,100 @@
+  "modalForm.attributes.select-components": "Selecciona los componentes",
+  "modalForm.component.header-create": "Crear un componente",
+  "modalForm.components.create-component.category.label": "Selecciona una categoría o ingresa un nombre para crear uno",
+  "modalForm.components.icon.label": "Ícono",
+  "modalForm.contentType.header-create": "Crear un Contenedor",
+  "modalForm.editCategory.base.name.description": "No se pueden usar espacios para nombrar una categoría",
+  "modalForm.header.categories": "Categorias",
+  "modalForm.header-edit": "Editar {name}",
+  "modalForm.singleType.header-create": "Crear un Contenedor",
+  "modalForm.sub-header.addComponentToDynamicZone": "Crear un componente al área dinámica",
+  "modalForm.sub-header.attribute.create": "Agrega un nuevo campo {type}",
+  "modalForm.sub-header.attribute.create.step": "Agrega un nuevo paso ({step}/2)",
+  "modalForm.sub-header.attribute.edit": "Editar {name}",
+  "modalForm.sub-header.chooseAttribute.collectionType": "Selecciona un campo para tu Contenedor",
+  "modalForm.sub-header.chooseAttribute.component": "Selecciona un campo para tu componente",
+  "modalForm.sub-header.chooseAttribute.singleType": "Selecciona un campo para tu Contenedor Único",
+  "modelPage.attribute.relation-polymorphic": "Relación (muchos a muchos)",
   "modelPage.attribute.relationWith": "Vinculación con",
-  "modelPage.contentHeader.emptyDescription.description": "No hay descripción para este Tipo de Contenido",
+  "modelPage.contentHeader.emptyDescription.description": "No hay una descripción para este Tipo de Contenido",
+  "none": "Ninguno",
+  "notification.info.autoreaload-disable": "Se requiere la característica de auto-recarga para usar este plugin. Comienza tu servidor con `strapi develop`",
+  "notification.info.creating.notSaved": "Por favor, guarda tus cambios antes de crear una nueva Colección o un componente",
   "plugin.description.long": "Modelice la estructura de datos de su API. Cree nuevos campos y relaciones en sólo un minuto. Los archivos se crean y actualizan automáticamente en el proyecto.",
   "plugin.description.short": "Modelice la estructura de datos de su API.",
   "popUpForm.navContainer.advanced": "Configuración avanzada",
   "popUpForm.navContainer.base": "Ajustes básicos",
-  "popUpWarning.bodyMessage.contentType.delete": "¿Está seguro de que desea eliminar este tipo de contenido?",
+  "popUpWarning.bodyMessage.cancel-modifications": "¿Estás seguro de desechar tus cambios?",
+  "popUpWarning.bodyMessage.cancel-modifications.with-components": "¿Estás seguro de desechar tus cambios? Has creado o modificado algunos componentes",
+  "popUpWarning.bodyMessage.category.delete": "¿Estás seguro de elminiar esta categoría? Todos sus componentes también serán elminados",
+  "popUpWarning.bodyMessage.component.delete": "¿Estás seguro de elminiar este componente?",
+  "popUpWarning.bodyMessage.contentType.delete": "¿Está seguro de que desea eliminar este Contenedor?",
```

Comment:
@waclock what u think about this?  "¿Está seguro de que desea eliminar este este tipo de contenido?"

Reply:
I'd change it to 2nd person:  "¿estás seguro que deseas eliminar este Contendor?", it'd align better with other delete messages as well. Good catch.

Reply:
"¿Estás seguro de que deseas eliminar este Contendor?". Missed a "de".

Reply:
> Is just the word " Contenedor" sounds strange, because is no a "contenedor" is a "content" with a "type"

Yes, I understand that in english, but in Spanish sound very strange and confuse, e.g.: "Agregar un nuevo Tipo de Contenido", means another thing.
One alternative is "Contenido".